### PR TITLE
More updates for translations

### PR DESCRIPTION
**Pull request by Lymkwi**
I also found a couple of strings here and there that I had put in the gettext call but written in english :sweat_smile:

### CODE REVIEW

File: locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -8,17 +8,59 @@
...
+#: insalan/cms/models.py:50
+msgid "Constante"
+msgstr "Constant"
+
+#: insalan/cms/models.py:51
+msgid "Constantes"
+msgstr "Constants"
+
+#: insalan/cms/tests.py:26
+msgid "Des constantes non définies sont utilisées: inconnue, random"
```
Comment (ShiroUsagi-san) [inline locale/en/LC_MESSAGES/django.po:61]: this is a hardcoded transaltion of variables names, is this correct?

**Review by ShiroUsagi-san — COMMENTED** [review]
one minor question but this is just tea time compared to the _payment branch_

File: locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (Lymkwi) [reply locale/en/LC_MESSAGES/django.po:61]: I thought about it and in my opinion if we are talking about variable names it should probably not be leaked/spewed out into an error destined to a user

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -60,28 +102,197 @@ msgstr "Partner/Sponsor name"
...
+#, python-format
+msgid "Transaction %(id)s en état invalide"
+msgstr "Transaction %(id)s in invalid state"
+
+#: insalan/payment/models.py:348
+msgid "Nombre d'un produit"
+msgstr "Product count"
+
+#: insalan/payment/models.py:349
+msgid "Nombres d'un produit"
```
Comment (Lugrim) [inline locale/en/LC_MESSAGES/django.po:260]: `Nombre de produits` ?

**Review by Lugrim — COMMENTED** [review]
A small wrong-sounding translation, but overall looks good to me

File: locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (Lymkwi) [reply locale/en/LC_MESSAGES/django.po:260]: More like `quantités d'un produit` tbh. It's the translation for `ProductCount`, which is just the medium model used for Many2Many between a transaction and products, so we can also store the amount of what products is bought.

I could instead have:
- singular: `Quantité d'un produit`
- plural: `Quantités de produits`

what do you think?

File: locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (Lymkwi) [reply locale/en/LC_MESSAGES/django.po:61]: Still, I will probably just ammend my commit to not translate the variable names and leave that to be fixed later.

File: locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (Lugrim) [reply locale/en/LC_MESSAGES/django.po:260]: Sounds way better

Comment (Lymkwi) [reply locale/en/LC_MESSAGES/django.po]: Done

**Review by Lugrim — APPROVED** [review]

**Review by ShiroUsagi-san — APPROVED** [review]

**Pull request merged**